src/SixLabors.Fonts/GlyphMetric.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.Primitives;
+
+namespace SixLabors.Fonts
+{
+    /// <summary>
+    /// Represents a <see cref="Glyph"/> metric.
+    /// </summary>
+    public struct GlyphMetric
```

Comment:
If you mark the struct as `readonly` this will prevent the compiler creating defensive copies. We generally ensure all structs implement `IEquatable<T>` also to allow efficient comparison in the future.

Reply:
Made struct readonly.

I don't see Equals ever being used for this struct though, as it concerns glyph layout info. For a low level structs like points and vectors I can see the use of having a type safe Equals method.